### DISCUSSION TITLE
Centralized status overview and Autoupdate from Stacks

### DISCUSSION THREAD

**Discussion by Eloooo-eu-org** [louislam/dockge#648] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

API, UI Feature

### 🔖 Feature description

It would be very nice if Dockge could display another Status "Buble" indicating the Status "Latest/Out-of-date" of the respective image used, besides the existing one indicating whether the Stack is "Active/Exited" in the sidebar.

It would also be important to have a new menu option ("Stacks" maybe? or some other name) to allow the selection and management multiple Stacks at once: Start/Stop/Update, and Autoupdate. The Autoupdate option, if implemented, would have a toggle for each Stack, to select which Stacks should autoupdate; and a separated area with fields to set up the  Timing of the updates (Daily/Weekly/Monthly, Date, Time etc).
 
I love Dockge and have a HomeLab with a lot of Stacks of no big importance, but that needs to be always up to date. So, I constantly have to open Portainer and check whether the containers are out-of-date or not, and open Dockge to update them one by one. It would be **really really nice**, if I could just configure it on Dockge and forget...

This request don't overlap with, but is somewhat dependent on and complements the following existing requests:
Show image update(s) available #239
Add Stack Automatisation #513

Thanks to everyone involved in this great software! Have a nice day!

### ✔️ Solution

1. Add UI elements to allow the user to select multiple stacks at onde and perform bulk actions on them.
2. Add the option to add the Stack to an automatic Autoupdate according to the defined time plan.
3.Add the same options to the Dockge API.

### ❓ Alternatives

containrrr/watchtower might be one alternative to achieve the same goal, but instead of adding another product (and complexity layer) to the mix, I wish Dockge could offer it natively.

### 📝 Additional Context

If more information is needed, I can gladly provide.

**Comment by zandadoum** [comment]
bulk selection (even over several server agents) to update would be great!
